packages/adapters-library/src/adapters/aave-v2/products/a-token/aaveV2ATokenAdapter.ts

```diff
@@ -1,15 +1,26 @@
 import { CacheToFile } from '../../../../core/decorators/cacheToFile'
+import { logger } from '../../../../core/utils/logger'
+import { Helpers } from '../../../../scripts/helpers'
+import { IProtocolAdapter } from '../../../../types/IProtocolAdapter'
 import {
   AssetType,
+  GetEventsInput,
+  GetPositionsInput,
+  GetTotalValueLockedInput,
+  MovementsByBlock,
   PositionType,
+  ProtocolAdapterParams,
   ProtocolDetails,
+  ProtocolPosition,
+  ProtocolTokenTvl,
+  UnwrapExchangeRate,
+  UnwrapInput,
```

Comment:
nit: All these imports are not required.

Reply:
Ignore this. It's better to go ahead. I'll see if I can introduce it as a mandatory lint error and correct them automatically, as other files are affected too.